src/data/finset/basic.lean

```diff
@@ -1823,6 +1823,14 @@ end
   to_finset l.reverse = l.to_finset :=
 to_finset_eq_of_perm _ _ (reverse_perm l)
 
+lemma to_finset_union (l l' : list α) :
+(l ∪ l').to_finset = l.to_finset ∪ l'.to_finset :=
+by {ext, simp}
+
+lemma to_finset_inter (l l' : list α) :
+(l ∩ l').to_finset = l.to_finset ∩ l'.to_finset :=
+by {ext, simp}
```

Comment:
style:
```suggestion
lemma to_finset_union (l l' : list α) : (l ∪ l').to_finset = l.to_finset ∪ l'.to_finset :=
by { ext, simp }

lemma to_finset_inter (l l' : list α) : (l ∩ l').to_finset = l.to_finset ∩ l'.to_finset :=
by { ext, simp }
```

Reply:
It probably makes sense to make these both `@[simp]` lemmas too.

Reply:
Thanks, I've made those changes